dashboard/app/controllers/plc/content_creator_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Plc::ContentCreatorController < ApplicationController
-  before_filter :require_levelbuilder_mode
-
   def show_courses_and_modules
+    authorize! :read, Plc::Course
```

Comment:
Admins can read/write courses, no one else can do anything to them right now. But seeing as how that will eventually change, I'll adjust to :manage